aem/aem_healthcheck_servlet.md

```diff
@@ -0,0 +1,49 @@
+   
+  
+## Trigger Adobe I/O Events health checks
+
+Once all your configurations are completed, in order to verify **and enable** 
```

Comment:
Why is this required for **enabling** the emission of events?

Reply:
to guarantee the Adobe I/O Events Providers Metadata are persisted in our control plane.

Reply:
Yes but even without the health check, the provider and corresponding event metadata entries will be persisted, right? My question/doubt that comes out of this line is whether running the health check a required step now?

Reply:
yes It is required.

Reply:
if the customers follows the installation step in the exact order, the provider metadata will already be persisted... but better to be safe than sorry...

Reply:
Ok